pkg/git/github/push.go

```diff
@@ -114,6 +114,9 @@ func (p *Push) IsBranch(branchToTry string) bool {
 
 // IsMaster detects if the branch is master.
 func (p *Push) IsMaster() bool {
+	if(p.Ref) == "master" {
+		return true;
```

Comment:
Can be simplified by `return p.Ref == "master" || p.Ref == "refs/heads/master"`

Reply:
check